app/scripts/metamask-controller.js

```diff
@@ -490,7 +490,7 @@ export default class MetamaskController extends EventEmitter {
     this.platform = opts.platform;
     this.notificationManager = opts.notificationManager;
     const initState = opts.initState || {};
-    const version = process.env.METAMASK_VERSION;
+    this.version = process.env.METAMASK_VERSION;
```

Comment:
Some stuff that used this `version` value was moved out of the constructor into a function. Also noticed a few other places `process.env.METAMASK_VERSION` was set, so I updated those to use the new class property I've created here.